circuit_knitting/cutting/cutqc/dynamic_definition.py

```diff
@@ -46,10 +46,17 @@
 from typing import Sequence, Any
 from qiskit import QuantumCircuit
 from qiskit.circuit import Qubit
+from qiskit.utils.deprecation import deprecate_func
 
 from .wire_cutting_post_processing import build, find_process_jobs
 
 
+@deprecate_func(
+    removal_timeline="Circuit knitting toolbox 0.8.0 release",
+    since="0.7.0",
+    package_name="cutqc",
```

Comment:
I _think_ `package_name` in all places should be `circuit-knitting-toolbox`.  For instance, on the qiskit `stable/0.46` branch, it's either `qiskit` or `qiskit-terra` everywhere.

Reply:
I think you're right, thanks.